src/demos/CartoonMirror.js

```diff
@@ -0,0 +1,38 @@
+import Camera from '../components/Camera';
+import useModel from '../hooks/useModel';
+import BlazePoseLoader from '../models/BlazePoseLoader';
+
+const style = {
+  position: "absolute",
+  top: 0,
+  left: 0,
+  right: 0,
+  zIndex: -1,
+  width: 160,
+  height: 120
+};
+
+const estimatePoseParam = {
+  maxPoses: 1,
+  flipHorizontal: true
+};
+
+const CartoonMirror = (props) => {
+  const detector = useModel(BlazePoseLoader, {backend: 'wasm', runtime: 'mediapipe'});
+
+  async function onEstimate(video) {
+    const hands = detector.current;
+    if (hands !== null) {
+      const poses = await hands.estimatePoses(video, estimatePoseParam);
+      console.log(poses);
+    }
+  }
+    
+  return (
+    <>
```

Comment:
What is this react fragment for? Given there is only one Camera component to return, the fragment is useless here, isn't it?